py4DSTEM/process/fit/fit.py

```diff
@@ -46,27 +54,39 @@ def fit_2D(function, data, data_mask=None, popt_guess=None):
                     same shape as data, containing the fit values
     """
     shape = data.shape
+    shape1D = [1,np.prod(shape)]
     x,y = np.arange(shape[0]),np.arange(shape[1])
+    x,y = np.linspace(0, 1, shape[0]),np.linspace(0, 1, shape[1])
```

Comment:
add a comment at the end of this line that x,y are normalized coordinates

Reply:
comment added!